types/kendo-ui/index.d.ts

```diff
@@ -743,6 +743,10 @@ declare namespace kendo.data {
         parentNode(): Node;
     }
 
+    class FileEntrty extends Node {
```

Comment:
`FileEntry`?

Reply:
@uniqueiniquity Thanks for noticing the typo. It's fixed now.